src/Pages/Profile/MemberView/Profile.jsx

```diff
@@ -40,7 +40,9 @@ export default class Profile extends Component {
           { title: 'Door Code', value: this.state.user.doorCode },
           { title: 'Joined Date', value: this.state.user.joinDate.slice(0, 10) },
           { title: 'Email', value: this.state.user.email },
-          { title: 'Membership Expiration', value: this.state.user.membershipValidUntil.slice(0, 10) },
+          { title: 'Membership Expiration', value: (this.props.user.accessLevel<0)? 
```

Comment:
no more hardcoded numbers. [we have enums for this](https://github.com/SCE-Development/Core-v4/blob/3c4ba959ba6a462c28bae5841594edf1ed625033/src/Enums.js#L28) so import them

Reply:
BRUH

Reply:
forgot to commit